tests/testsuite/version.rs

```diff
@@ -1,21 +1,30 @@
 //! Tests for displaying the cargo version.
 
-use cargo_test_support::{cargo_process, project};
+use cargo_test_support::{cargo_process, project, str};
 
 #[cargo_test]
 fn simple() {
     let p = project().build();
 
     p.cargo("version")
-        .with_stdout_data(&format!("cargo {}\n", cargo::version()))
+        .with_stdout_data(str![[r#"
+cargo [CARGO_VERSION]
+
+"#]])
```

Comment:
ditto about maintaining the old expected-value and just updating the assert method